salt/states/postgres_group.py

```diff
@@ -71,7 +65,17 @@ def present(
         Is the group allowed to create other roles/users
 
     encrypted
-        Should the password be encrypted in the system catalog?
+        How the password should be stored.
+
+        If encrypted is ``None``, ``True``, or ``md5``, it will use
+        PostgreSQL's MD5 algorithm.
+
+        If encrypted is ``False``, it will be stored in plaintext.
+
+        If encrypted is ``scram-sha-256``, it will use the algorithm described
+        in RFC 7677.
+
+        # TODO: versionadded for this?
```

Comment:
I wasn't sure how you indicate versions when parameter options are added, rather than the whole parameter.